src/integration-test/java/io/aiven/connect/jdbc/pg/PgSqlSinkConnectorIT.java

```diff
@@ -0,0 +1,239 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.connect.jdbc.pg;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.Future;
+
+import org.apache.kafka.clients.producer.RecordMetadata;
+
+import io.aiven.connect.jdbc.JdbcSinkConnector;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PgSqlSinkConnectorIT extends AbstractPgSqlAwareIT {
+
+    static final Schema VALUE_SCHEMA = new Schema.Parser().parse(
+            "{"
+                    + "\"type\":\"record\","
+                    + "\"name\":\"pg_sql_types\","
+                    + "\"fields\":"
+                    + "["
+                    + "{\"name\":\"id\",\"type\":\"int\"},"
+                    + "{\"name\":\"json_value\", \"type\":\"string\"},"
+                    + "{\"name\":\"jsonb_value\", \"type\":\"string\"}"
+                    + "]}");
+
+    static final String SELECT_QUERY = "SELECT id, json_value, jsonb_value FROM " + TEST_TOPIC_NAME + " ORDER BY id";
+
+    private final ArrayDeque<String> createdConnectors = new ArrayDeque<>();
+
+    private final Random partitionRnd = new Random();
+
+    @BeforeEach
+    void createDbTable() throws Exception {
+        try (final var stm = pgConnection.createStatement()) {
+            final var createTableSql =
+                    String.format(
+                            "CREATE TABLE %s ("
+                                    + " id INT PRIMARY KEY NOT NULL,"
+                                    + " json_value JSON NOT NULL,"
+                                    + " jsonb_value JSONB NOT NULL"
+                                    + " )",
+                            TEST_TOPIC_NAME
+                    );
+            LOGGER.info("Create table: {}", createTableSql);
+            stm.execute(createTableSql);
+        }
+    }
+
+    @AfterEach
+    void deleteConnector() throws Exception {
+        final var previousConnector = createdConnectors.poll();
+        jdbcConnectService.deleteConnector(previousConnector);
+        try (final var stm = pgConnection.createStatement()) {
+            final var dropTableSql = "DROP TABLE " + TEST_TOPIC_NAME;
+            LOGGER.info("Drop table: {}", dropTableSql);
+            stm.executeUpdate(dropTableSql);
+        }
+    }
+
+    /**
+     * Test for possible PgSQL types like: JSON, JSONB, ARRAY, UUID and etc.
```

Comment:
JSON types are tested, what about others?

Reply:
So far this test covers only JSON/JSONB cast types

Reply:
I will add them one by one

Reply:
Either the code or comment should be changed then :)

Reply:
kk

Reply:
I added UUID :-) will chnage comment

Reply:
I guess this still needs doing?